codice/ddf
# DDF-3990 Revert ECQL mapper visitor change to fix CSW Store delete

#### What does this PR do?
Revert ECQL mapper visitor change that was needed previously for an earlier version of the ECQL classes.
#### Who is reviewing it? 

@emmberk @peterhuffer 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Ask 2 committers to review/merge the PR and tag them here.

@rzwiefel
@vinamartin
#### How should this be tested?

CI with new unit test should be sufficient. 
If manual testing, install two ddfs with registry and verify you can publish and then delete one node from the other.
#### Any background context you want to provide?
DDF-3071 upgrade filter schema to 1.1.3
#### What are the relevant tickets?
[DDF-3990](https://codice.atlassian.net/browse/DDF-3990)
#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
